presto-native-execution/presto_cpp/main/SessionProperties.cpp

```diff
@@ -567,12 +567,12 @@ SessionProperties::SessionProperties() {
       std::to_string(c.maxOutputBatchRows()));
 
   addSessionProperty(
-      kRowSizeTrackingEnabled,
-      "A fallback for average row size estimate when not supported for certain readers. Turned on by default.",
-      BOOLEAN(),
+      kRowSizeTrackingMode,
+      "Enable (reader) row size tracker as a fallback to file level row size estimates.",
+      INTEGER(),
       true,
-      QueryConfig::kRowSizeTrackingEnabled,
-      std::to_string(c.rowSizeTrackingEnabled()));
+      QueryConfig::kRowSizeTrackingMode,
+      std::to_string(static_cast<int32_t>(c.rowSizeTrackingMode())));
```

Comment:
@tanjialiang : This is just going to show 1, 2, 3 and not the string versions of the enums. We should ask Velox team to expose the string names of the enums (they have a macro for it) which we can use here.

https://github.com/facebookincubator/velox/pull/15111#pullrequestreview-3362980946

Reply:
yeah we can do a followup on this